Tribler/Core/defaults.py

```diff
@@ -163,6 +164,26 @@
 sessdefaults['http_api']['enabled'] = False
 sessdefaults['http_api']['port'] = -1
 
+# Credit mining config
+sessdefaults['credit_mining'] = OrderedDict()
+sessdefaults['credit_mining']['enabled'] = True
```

Comment:
There will be changes in GUI and Core as well if I do this. Considering this PR is big enough already, I can do this in next PR.